src/commands/dailyCheckIn/scheduleCheckIn.js

```diff
@@ -0,0 +1,186 @@
+            }
+            catch {
+                await interaction.editReply({
+                    ephemeral: true,
+                    content: 'Invalid time'
+                });
+                return;
+            }
+
+            // calculate difference from local time zone to UTC-0.
+            // This is to ensure that everyone's notifications are timed properly 
+            // regardless of time zone
+            const utcDay = dayjs(interaction?.createdAt).utc().day();
+            const localDay = dayjs().hour(timeHours).utc().day();
+            const dayDiff = localDay - utcDay;
+            const utcHour = dayjs().hour(timeHours).utc().hour();
+            const utcMin = dayjs().hour(timeHours).minute(timeMinutes).utc().minute();
+            // dayjs().day(1).hour(17)
+
+            const utcDays = parsedDays.map(d => {
+                // shift the day depending on time zone offset
+                const rawDayIndex = validDays.indexOf(d) + dayDiff;
+
+                // force index within bounds of validDays[]
+                const dayIndex = rawDayIndex % validDays.length < 0 ?
+                    validDays.length + (rawDayIndex % validDays.length) :
+                    rawDayIndex % validDays.length;
+                return validDays[dayIndex];
+            })
+
+            const formattedTimeMin = timeMinutes.toString().length === 1 ? `0${timeMinutes}` : timeMinutes;
+
+
+            // update the database
+
+            // clear current user's entries
+
+            Object.keys(fakeScheduleEntry).forEach(day => {
+                delete fakeScheduleEntry[day][userId]
+            });
+
+            // update user's entries
+            utcDays.forEach((d, i) => {
+                fakeScheduleEntry[d][userId] = {
+                    id: userId,
+                    tag: userTag,
+                    utcTime: [utcHour, utcMin],
+                    localTime: `${parsedDays[i]} ${timeHours}:${formattedTimeMin}`
+                }
+            });
+
+            // respond to the user to confirm schedule
+            const daysResponse = daily ? 'every day' : `[${parsedDays.join(', ')}]`;
+
+            let reply = ``;
+            reply += `Check ins scheduled for ${daysResponse} at ${timeHours}:${formattedTimeMin} (24hr time)\n`
+            reply += `\n[debug]\n`;
+            reply += `\`\`\`json\n${JSON.stringify(fakeScheduleEntry, undefined, 2)}\`\`\``
+            await interaction.editReply({
+                ephemeral: true,
+                content: reply
+            });
+        }
+        catch {
+            await interaction.editReply({
+                ephemeral: true,
+                content: `issue running command`
+            });
+        }
+    },
+    name: 'schedule-check-in',
+    description: 'Schedule a day and time to be notified',
+    options: [
+        {
+            name: 'days',
+            description: 'List of days. Ex: "monday wednesday friday" or "daily"',
```

Comment:
I don't love this method of specifying days; too error-prone and not a great interface. But I recognize that this is possibly a limitation of this interface so let's ignore my misgivings for now.